### PR TITLE
Add `mount` tag

### PR DESCRIPTION
**Pull request by jacksleight**
This PR adds a `mount` tag which returns the mount URL of a collection, useful if you want to dynamically link to a collection's main page:

```html
<a href="{{ mount:blog }}">Read Our Blog</a>
```

If this might be accepted I can do a docs PR as well.

### CODE REVIEW

**Comment by robdekort** [comment]
This nice. This way I can drop the custom one in Peak!

**Comment by robdekort** [comment]
Heads up: I don't think this returns a localized URL.

File: src/Tags/Mount.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return $this->mount($this->params->get('handle'));
+    }
+
+    private function mount($handle)
+    {
+        if (! $collection = Collection::find($handle)) {
+            return;
+        }
+
+        return $collection->url();
```
Comment (robdekort) [inline src/Tags/Mount.php:31]: use Statamic\Facades\Site;

`$collection->url(Site::current()->handle())`

To get the localized version.

Comment (jacksleight) [reply src/Tags/Mount.php:31]: Cheers Rob, I'll update it.

Comment (jasonvarga) [reply src/Tags/Mount.php]: I've updated it and the test.

**Review by jasonvarga — APPROVED** [review]

File: src/Tags/Mount.php (hunk 1 above)
Comment (jacksleight) [reply src/Tags/Mount.php]: Ah thanks!

Comment (robdekort) [reply src/Tags/Mount.php]: Lovely!

**Pull request merged**